packages/composites/the-graph/src/config.ts

```diff
@@ -0,0 +1,31 @@
+import { Config as DefaultConfig } from "@chainlink/types"
+import { DexSubgraph } from "./types"
+import { uniswapSubgraph } from "./methods/prices/dex"
+
+export type Config = DefaultConfig & {
+  infuraHost: string,
+  dexSubgraphs: {
+    [T: string]: DexSubgraph
+  }
+}
+
+export const INFURA_HOST = process.env.INFURA_HOST || ""
+export const USDT_USD_AGGREGATOR_V3_ADDRESS = process.env.AGGREGATOR_V3_ADDRESS || "0x3E7d1eAB13ad0104d2750B8863b489D65364e32D"
+
+export const PRICE = "price"
+
+export const USD = "USD"
+export const ETH = "ETH"
+export const WETH = "WETH"
+export const USDT = "USDT"
```

Comment:
We should be able to get rid of this to make it as generic as possible. See my other comments.

Reply:
I've left `WETH` as I'm using it as a default value